src/openzaak/conf/includes/base.py

```diff
@@ -693,6 +693,11 @@
     "mozilla_django_oidc_db.backends.OIDCAuthenticationBackend",
 ]
 
+# if DISABLE_2FA is true, fill the MAYKIN_2FA_ALLOW_MFA_BYPASS_BACKENDS with all
+# configured AUTHENTICATION_BACKENDS and thus disabeling the entire 2FA chain
+if config("DISABLE_2FA", default=True):  # pragma: no cover
```

Comment:
```suggestion
if config("DISABLE_2FA", default=False):  # pragma: no cover
```

Let's make it more secure by default